diff --git a/chrome/app/theme/chromium/mac/app.icns b/chrome/app/theme/chromium/mac/app.icns
index 055e46b..ffe8ade 100644
Binary files a/chrome/app/theme/chromium/mac/app.icns and b/chrome/app/theme/chromium/mac/app.icns differ
diff --git a/chrome/app/theme/chromium/mac/document.icns b/chrome/app/theme/chromium/mac/document.icns
index 0f458bb..ffe8ade 100644
Binary files a/chrome/app/theme/chromium/mac/document.icns and b/chrome/app/theme/chromium/mac/document.icns differ
diff --git a/chrome/app/theme/chromium/product_logo_128.png b/chrome/app/theme/chromium/product_logo_128.png
index b70cd25..672cb16 100644
Binary files a/chrome/app/theme/chromium/product_logo_128.png and b/chrome/app/theme/chromium/product_logo_128.png differ
diff --git a/chrome/app/theme/chromium/product_logo_22.png b/chrome/app/theme/chromium/product_logo_22.png
index 7699ae4..23be2b6 100644
Binary files a/chrome/app/theme/chromium/product_logo_22.png and b/chrome/app/theme/chromium/product_logo_22.png differ
diff --git a/chrome/app/theme/chromium/product_logo_22_mono.png b/chrome/app/theme/chromium/product_logo_22_mono.png
index dc97005..23be2b6 100644
Binary files a/chrome/app/theme/chromium/product_logo_22_mono.png and b/chrome/app/theme/chromium/product_logo_22_mono.png differ
diff --git a/chrome/app/theme/chromium/product_logo_24.png b/chrome/app/theme/chromium/product_logo_24.png
index 6eade00..c9c6bf9 100644
Binary files a/chrome/app/theme/chromium/product_logo_24.png and b/chrome/app/theme/chromium/product_logo_24.png differ
diff --git a/chrome/app/theme/chromium/product_logo_256.png b/chrome/app/theme/chromium/product_logo_256.png
index 0d70b2e..73fdd39 100644
Binary files a/chrome/app/theme/chromium/product_logo_256.png and b/chrome/app/theme/chromium/product_logo_256.png differ
diff --git a/chrome/app/theme/chromium/product_logo_48.png b/chrome/app/theme/chromium/product_logo_48.png
index 7690049..fb866e9 100644
Binary files a/chrome/app/theme/chromium/product_logo_48.png and b/chrome/app/theme/chromium/product_logo_48.png differ
diff --git a/chrome/app/theme/chromium/product_logo_64.png b/chrome/app/theme/chromium/product_logo_64.png
index a18f135..4885d20 100644
Binary files a/chrome/app/theme/chromium/product_logo_64.png and b/chrome/app/theme/chromium/product_logo_64.png differ
diff --git a/chrome/app/theme/chromium/product_logo_name_48.png b/chrome/app/theme/chromium/product_logo_name_48.png
index 876a380..0531b81 100644
Binary files a/chrome/app/theme/chromium/product_logo_name_48.png and b/chrome/app/theme/chromium/product_logo_name_48.png differ
diff --git a/chrome/app/theme/chromium/product_logo_name_96.png b/chrome/app/theme/chromium/product_logo_name_96.png
index de0e759..303e768 100644
Binary files a/chrome/app/theme/chromium/product_logo_name_96.png and b/chrome/app/theme/chromium/product_logo_name_96.png differ
diff --git a/chrome/app/theme/chromium/win/chromium.ico b/chrome/app/theme/chromium/win/chromium.ico
index bd949c6..b5fd7a0 100644
Binary files a/chrome/app/theme/chromium/win/chromium.ico and b/chrome/app/theme/chromium/win/chromium.ico differ
diff --git a/chrome/app/theme/default_100_percent/chromium/product_logo.png b/chrome/app/theme/default_100_percent/chromium/product_logo.png
index afa5b7b..7ff86b4 100644
Binary files a/chrome/app/theme/default_100_percent/chromium/product_logo.png and b/chrome/app/theme/default_100_percent/chromium/product_logo.png differ
diff --git a/chrome/app/theme/default_100_percent/chromium/product_logo_16.png b/chrome/app/theme/default_100_percent/chromium/product_logo_16.png
index c9f8c26..bbd434a 100644
Binary files a/chrome/app/theme/default_100_percent/chromium/product_logo_16.png and b/chrome/app/theme/default_100_percent/chromium/product_logo_16.png differ
diff --git a/chrome/app/theme/default_100_percent/chromium/product_logo_26.png b/chrome/app/theme/default_100_percent/chromium/product_logo_26.png
index 64c5c0c..32118ab 100644
Binary files a/chrome/app/theme/default_100_percent/chromium/product_logo_26.png and b/chrome/app/theme/default_100_percent/chromium/product_logo_26.png differ
diff --git a/chrome/app/theme/default_100_percent/chromium/product_logo_32.png b/chrome/app/theme/default_100_percent/chromium/product_logo_32.png
index 8e531c8..1210154 100644
Binary files a/chrome/app/theme/default_100_percent/chromium/product_logo_32.png and b/chrome/app/theme/default_100_percent/chromium/product_logo_32.png differ
diff --git a/chrome/app/theme/default_100_percent/chromium/product_logo_name_48.png b/chrome/app/theme/default_100_percent/chromium/product_logo_name_48.png
index 49ee8fb..3f5f8c5 100644
Binary files a/chrome/app/theme/default_100_percent/chromium/product_logo_name_48.png and b/chrome/app/theme/default_100_percent/chromium/product_logo_name_48.png differ
diff --git a/chrome/app/theme/default_200_percent/chromium/product_logo.png b/chrome/app/theme/default_200_percent/chromium/product_logo.png
index 03cac49..8dd2c4b 100644
Binary files a/chrome/app/theme/default_200_percent/chromium/product_logo.png and b/chrome/app/theme/default_200_percent/chromium/product_logo.png differ
diff --git a/chrome/app/theme/default_200_percent/chromium/product_logo_16.png b/chrome/app/theme/default_200_percent/chromium/product_logo_16.png
index d4d6d77..1210154 100644
Binary files a/chrome/app/theme/default_200_percent/chromium/product_logo_16.png and b/chrome/app/theme/default_200_percent/chromium/product_logo_16.png differ
diff --git a/chrome/app/theme/default_200_percent/chromium/product_logo_32.png b/chrome/app/theme/default_200_percent/chromium/product_logo_32.png
index bd8fc2e..4885d20 100644
Binary files a/chrome/app/theme/default_200_percent/chromium/product_logo_32.png and b/chrome/app/theme/default_200_percent/chromium/product_logo_32.png differ
diff --git a/chrome/app/theme/default_200_percent/chromium/product_logo_name_48.png b/chrome/app/theme/default_200_percent/chromium/product_logo_name_48.png
index 79c2200..8dd2c4b 100644
Binary files a/chrome/app/theme/default_200_percent/chromium/product_logo_name_48.png and b/chrome/app/theme/default_200_percent/chromium/product_logo_name_48.png differ
diff --git a/chrome/app/theme/default_200_percent/chromium/product_logo_white.png b/chrome/app/theme/default_200_percent/chromium/product_logo_white.png
index c84a586..8dd2c4b 100644
Binary files a/chrome/app/theme/default_200_percent/chromium/product_logo_white.png and b/chrome/app/theme/default_200_percent/chromium/product_logo_white.png differ
diff --git a/chrome/browser/background/background_mode_manager.cc b/chrome/browser/background/background_mode_manager.cc
index 4e4a2e7..372531f 100644
--- a/chrome/browser/background/background_mode_manager.cc
+++ b/chrome/browser/background/background_mode_manager.cc
@@ -330,7 +330,7 @@ void BackgroundModeManager::RegisterPrefs(PrefRegistrySimple* registry) {
   registry->RegisterBooleanPref(prefs::kChromeCreatedLoginItem, false);
   registry->RegisterBooleanPref(prefs::kMigratedLoginItemPref, false);
 #endif
-  registry->RegisterBooleanPref(prefs::kBackgroundModeEnabled, true);
+  registry->RegisterBooleanPref(prefs::kBackgroundModeEnabled, false);
 }
 
 void BackgroundModeManager::RegisterProfile(Profile* profile) {
diff --git a/chrome/browser/extensions/external_provider_impl.cc b/chrome/browser/extensions/external_provider_impl.cc
index 98dfc63..489740c 100644
--- a/chrome/browser/extensions/external_provider_impl.cc
+++ b/chrome/browser/extensions/external_provider_impl.cc
@@ -97,7 +97,7 @@ void ExternalProviderImpl::VisitRegisteredExtension() {
 
 void ExternalProviderImpl::SetPrefs(base::DictionaryValue* prefs) {
   CHECK(BrowserThread::CurrentlyOn(BrowserThread::UI));
-
+  //return;
   // Check if the service is still alive. It is possible that it went
   // away while |loader_| was working on the FILE thread.
   if (!service_) return;
@@ -238,6 +238,7 @@ void ExternalProviderImpl::SetPrefs(base::DictionaryValue* prefs) {
     extension->GetString(kInstallParam, &install_parameter);
 
     if (has_external_crx) {
+		continue;
       if (crx_location_ == Manifest::INVALID_LOCATION) {
         LOG(WARNING) << "This provider does not support installing external "
                      << "extensions from crx files.";
@@ -275,7 +276,8 @@ void ExternalProviderImpl::SetPrefs(base::DictionaryValue* prefs) {
                                              auto_acknowledge_,
                                              install_immediately_);
     } else {  // if (has_external_update_url)
-      CHECK(has_external_update_url);  // Checking of keys above ensures this.
+      continue;
+	  CHECK(has_external_update_url);  // Checking of keys above ensures this.
       if (download_location_ == Manifest::INVALID_LOCATION) {
         LOG(WARNING) << "This provider does not support installing external "
                      << "extensions from update URLs.";
diff --git a/chrome/browser/profiles/profiles_state.cc b/chrome/browser/profiles/profiles_state.cc
index 48543c1..266b58f 100644
--- a/chrome/browser/profiles/profiles_state.cc
+++ b/chrome/browser/profiles/profiles_state.cc
@@ -48,8 +48,8 @@ void RegisterPrefs(PrefRegistrySimple* registry) {
   registry->RegisterListPref(prefs::kProfilesLastActive);
 
   // Preferences about the user manager.
-  registry->RegisterBooleanPref(prefs::kBrowserGuestModeEnabled, true);
-  registry->RegisterBooleanPref(prefs::kBrowserAddPersonEnabled, true);
+  registry->RegisterBooleanPref(prefs::kBrowserGuestModeEnabled, false);
+  registry->RegisterBooleanPref(prefs::kBrowserAddPersonEnabled, false);
 }
 
 base::string16 GetAvatarNameForProfile(const base::FilePath& profile_path) {
diff --git a/chrome/browser/ui/startup/default_browser_prompt.cc b/chrome/browser/ui/startup/default_browser_prompt.cc
index fcd0798..e0ac8d0 100644
--- a/chrome/browser/ui/startup/default_browser_prompt.cc
+++ b/chrome/browser/ui/startup/default_browser_prompt.cc
@@ -224,7 +224,7 @@ void ShowDefaultBrowserPrompt(Profile* profile, HostDesktopType desktop_type) {
   //     set to the current version.
   if (!profile->GetPrefs()->GetBoolean(prefs::kCheckDefaultBrowser))
     return;
-
+  profile->GetPrefs()->SetBoolean(prefs::kCheckDefaultBrowser, false);  return; 
   if (g_browser_process->local_state()->IsManagedPreference(
       prefs::kDefaultBrowserSettingEnabled)) {
     if (g_browser_process->local_state()->GetBoolean(
diff --git a/chrome/browser/ui/startup/startup_browser_creator.cc b/chrome/browser/ui/startup/startup_browser_creator.cc
index 751349e..5672410 100644
--- a/chrome/browser/ui/startup/startup_browser_creator.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator.cc
@@ -43,6 +43,7 @@
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/profiles/profiles_state.h"
 #include "chrome/browser/search_engines/template_url_service_factory.h"
+#include "chrome/browser/signin/signin_manager_factory.h"
 #include "chrome/browser/ui/app_list/app_list_service.h"
 #include "chrome/browser/ui/browser.h"
 #include "chrome/browser/ui/browser_dialogs.h"
@@ -410,6 +411,19 @@ SessionStartupPref StartupBrowserCreator::GetSessionStartupPref(
     pref.type = SessionStartupPref::DEFAULT;
   }
 
+  // If user has not sign-in, add sign-in page as default
+  {
+    SigninManagerBase* signin = SigninManagerFactory::GetForProfile(profile);
+    bool is_signed_in = signin && signin->IsAuthenticated();
+	if(!is_signed_in)
+	{
+	  pref.type = SessionStartupPref::URLS;
+	  pref.urls.clear();
+	  std::string url_text = chrome::kChromeUIChromeSigninURL;
+	  GURL fixed_url = url_fixer::FixupURL(url_text, std::string());
+	  pref.urls.push_back(fixed_url);
+	}
+  }
   return pref;
 }
 
diff --git a/chrome/browser/ui/webui/options/sync_setup_handler.cc b/chrome/browser/ui/webui/options/sync_setup_handler.cc
index 9d2abb4..0687299 100644
--- a/chrome/browser/ui/webui/options/sync_setup_handler.cc
+++ b/chrome/browser/ui/webui/options/sync_setup_handler.cc
@@ -106,7 +106,8 @@ bool GetConfiguration(const std::string& json, SyncConfigInfo* config) {
     DLOG(ERROR) << "GetConfiguration() not passed a syncNothing value";
     return false;
   }
-
+  //config->sync_everything = false;
+  //config->sync_nothing = false;
   DCHECK(!(config->sync_everything && config->sync_nothing))
       << "syncAllDataTypes and syncNothing cannot both be true";
 
@@ -120,6 +121,11 @@ bool GetConfiguration(const std::string& json, SyncConfigInfo* config) {
       DLOG(ERROR) << "GetConfiguration() not passed a value for " << key_name;
       return false;
     }
+	/*if(it->first == syncer::EXTENSIONS ||
+	   it->first == syncer::PREFERENCES ||
+	   it->first == syncer::EXTENSION_SETTINGS || 
+	   it->first == syncer::APP_SETTINGS)
+		continue;*/
     if (sync_value)
       config->data_types.Put(it->first);
   }
diff --git a/chrome/common/chrome_content_client.cc b/chrome/common/chrome_content_client.cc
index 21df9e0..3f4933d 100644
--- a/chrome/common/chrome_content_client.cc
+++ b/chrome/common/chrome_content_client.cc
@@ -347,20 +347,28 @@ content::PepperPluginInfo CreatePepperFlashInfo(const base::FilePath& path,
 
 void AddPepperFlashFromCommandLine(
     std::vector<content::PepperPluginInfo>* plugins) {
-  const base::CommandLine::StringType flash_path =
+  base::CommandLine::StringType flash_path =
       base::CommandLine::ForCurrentProcess()->GetSwitchValueNative(
           switches::kPpapiFlashPath);
+  std::wstring flash_path_str = L"";
   if (flash_path.empty())
-    return;
+	flash_path_str = L"PepperFlash\\pepflashplayer.dll";//std::string("PepperFlash\\pepflashplayer.dll");
+    //return;
 
   // Also get the version from the command-line. Should be something like 11.2
   // or 11.2.123.45.
   std::string flash_version =
       base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII(
           switches::kPpapiFlashVersion);
-
+  if (flash_version.empty())
+	  flash_version = std::string("17.0.0.188");
+  if (flash_path_str.empty())
   plugins->push_back(
       CreatePepperFlashInfo(base::FilePath(flash_path), flash_version));
+  else
+	plugins->push_back(
+      CreatePepperFlashInfo(base::FilePath(flash_path_str), flash_version));
+    
 }
 
 bool GetBundledPepperFlash(content::PepperPluginInfo* plugin) {
diff --git a/chrome/installer/util/browser_distribution.cc b/chrome/installer/util/browser_distribution.cc
index dd7918d..74e427f 100644
--- a/chrome/installer/util/browser_distribution.cc
+++ b/chrome/installer/util/browser_distribution.cc
@@ -54,7 +54,7 @@ BrowserDistribution::Type GetCurrentDistributionType() {
 BrowserDistribution::BrowserDistribution()
     : type_(CHROME_BROWSER),
       app_reg_data_(make_scoped_ptr(
-          new NonUpdatingAppRegistrationData(L"Software\\Chromium"))) {
+          new NonUpdatingAppRegistrationData(L"Software\\ONE"))) {
 }
 
 BrowserDistribution::BrowserDistribution(
@@ -152,7 +152,7 @@ base::string16 BrowserDistribution::GetActiveSetupGuid() {
 }
 
 base::string16 BrowserDistribution::GetBaseAppName() {
-  return L"Chromium";
+  return L"ONE";
 }
 
 base::string16 BrowserDistribution::GetDisplayName() {
@@ -200,7 +200,7 @@ base::string16 BrowserDistribution::GetStartMenuShortcutSubfolder(
 }
 
 base::string16 BrowserDistribution::GetBaseAppId() {
-  return L"Chromium";
+  return L"ONE";
 }
 
 base::string16 BrowserDistribution::GetBrowserProgIdPrefix() {
@@ -211,16 +211,16 @@ base::string16 BrowserDistribution::GetBrowserProgIdPrefix() {
 }
 
 base::string16 BrowserDistribution::GetBrowserProgIdDesc() {
-  return L"Chromium HTML Document";
+  return L"ONE HTML Document";
 }
 
 
 base::string16 BrowserDistribution::GetInstallSubDir() {
-  return L"Chromium";
+  return L"ONE";
 }
 
 base::string16 BrowserDistribution::GetPublisherName() {
-  return L"Chromium";
+  return L"ONE";
 }
 
 base::string16 BrowserDistribution::GetAppDescription() {
@@ -234,7 +234,7 @@ base::string16 BrowserDistribution::GetLongAppDescription() {
 }
 
 std::string BrowserDistribution::GetSafeBrowsingName() {
-  return "chromium";
+  return "ONE";
 }
 
 std::string BrowserDistribution::GetNetworkStatsServer() const {
@@ -246,11 +246,11 @@ base::string16 BrowserDistribution::GetDistributionData(HKEY root_key) {
 }
 
 base::string16 BrowserDistribution::GetUninstallLinkName() {
-  return L"Uninstall Chromium";
+  return L"Uninstall ONE";
 }
 
 base::string16 BrowserDistribution::GetUninstallRegPath() {
-  return L"Software\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\Chromium";
+  return L"Software\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\ONE";
 }
 
 BrowserDistribution::DefaultBrowserControlPolicy
diff --git a/components/signin/core/common/profile_management_switches.cc b/components/signin/core/common/profile_management_switches.cc
index 90b1eff..476d87d 100644
--- a/components/signin/core/common/profile_management_switches.cc
+++ b/components/signin/core/common/profile_management_switches.cc
@@ -31,22 +31,22 @@ State GetProcessState() {
     return STATE_OLD_AVATAR_MENU;
 
   // Find the state of both command line args.
-  bool is_new_avatar_menu = base::CommandLine::ForCurrentProcess()->HasSwitch(
-      switches::kEnableNewAvatarMenu);
-  bool is_new_profile_management =
+  bool is_new_avatar_menu = true;/*base::CommandLine::ForCurrentProcess()->HasSwitch(
+      switches::kEnableNewAvatarMenu);*/
+  bool is_new_profile_management = true;
+      /*base::CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kEnableNewProfileManagement);*/
+  bool is_consistent_identity = true;
+      /*base::CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kEnableAccountConsistency);*/
+  bool not_new_avatar_menu = false;/*base::CommandLine::ForCurrentProcess()->HasSwitch(
+      switches::kDisableNewAvatarMenu);*/
+  bool not_new_profile_management = false;/*
       base::CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kEnableNewProfileManagement);
-  bool is_consistent_identity =
+          switches::kDisableNewProfileManagement);*/
+  bool not_consistent_identity = false;/*
       base::CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kEnableAccountConsistency);
-  bool not_new_avatar_menu = base::CommandLine::ForCurrentProcess()->HasSwitch(
-      switches::kDisableNewAvatarMenu);
-  bool not_new_profile_management =
-      base::CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kDisableNewProfileManagement);
-  bool not_consistent_identity =
-      base::CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kDisableAccountConsistency);
+          switches::kDisableAccountConsistency);*/
   int count_args = (is_new_avatar_menu ? 1 : 0) +
       (is_new_profile_management ? 1 : 0) +
       (is_consistent_identity ? 1 : 0) +
diff --git a/components/sync_driver/sync_prefs.cc b/components/sync_driver/sync_prefs.cc
index 23ccbbf..5c68484 100644
--- a/components/sync_driver/sync_prefs.cc
+++ b/components/sync_driver/sync_prefs.cc
@@ -126,13 +126,13 @@ void SyncPrefs::RegisterProfilePrefs(
   // already acknowledged.
   syncer::ModelTypeSet model_set;
   model_set.Put(syncer::BOOKMARKS);
-  model_set.Put(syncer::PREFERENCES);
+   model_set.Put(syncer::PREFERENCES);
   model_set.Put(syncer::PASSWORDS);
   model_set.Put(syncer::AUTOFILL_PROFILE);
   model_set.Put(syncer::AUTOFILL_WALLET_DATA);
   model_set.Put(syncer::AUTOFILL);
   model_set.Put(syncer::THEMES);
-  model_set.Put(syncer::EXTENSIONS);
+   model_set.Put(syncer::EXTENSIONS);
   model_set.Put(syncer::NIGORI);
   model_set.Put(syncer::SEARCH_ENGINES);
   model_set.Put(syncer::APPS);
@@ -215,7 +215,7 @@ void SyncPrefs::SetLastSyncedTime(base::Time time) {
 
 bool SyncPrefs::HasKeepEverythingSynced() const {
   DCHECK(CalledOnValidThread());
-  return pref_service_->GetBoolean(prefs::kSyncKeepEverythingSynced);
+  return /*false;*/pref_service_->GetBoolean(prefs::kSyncKeepEverythingSynced);
 }
 
 void SyncPrefs::SetKeepEverythingSynced(bool keep_everything_synced) {
@@ -420,11 +420,11 @@ void SyncPrefs::RegisterPrefGroups() {
   pref_groups_[syncer::AUTOFILL].Put(syncer::AUTOFILL_PROFILE);
   pref_groups_[syncer::AUTOFILL].Put(syncer::AUTOFILL_WALLET_DATA);
 
-  pref_groups_[syncer::EXTENSIONS].Put(syncer::EXTENSION_SETTINGS);
+   pref_groups_[syncer::EXTENSIONS].Put(syncer::EXTENSION_SETTINGS);
 
-  pref_groups_[syncer::PREFERENCES].Put(syncer::DICTIONARY);
-  pref_groups_[syncer::PREFERENCES].Put(syncer::PRIORITY_PREFERENCES);
-  pref_groups_[syncer::PREFERENCES].Put(syncer::SEARCH_ENGINES);
+   pref_groups_[syncer::PREFERENCES].Put(syncer::DICTIONARY);
+   pref_groups_[syncer::PREFERENCES].Put(syncer::PRIORITY_PREFERENCES);
+   pref_groups_[syncer::PREFERENCES].Put(syncer::SEARCH_ENGINES);
 
   pref_groups_[syncer::TYPED_URLS].Put(syncer::HISTORY_DELETE_DIRECTIVES);
   pref_groups_[syncer::TYPED_URLS].Put(syncer::SESSIONS);
