diff --git a/components/signin/core/common/profile_management_switches.cc b/components/signin/core/common/profile_management_switches.cc
index 2785cf4..5459598 100644
--- a/components/signin/core/common/profile_management_switches.cc
+++ b/components/signin/core/common/profile_management_switches.cc
@@ -25,24 +25,24 @@ enum State {
 
 State GetProcessState() {
   // Find the state of both command line args.
-  bool is_new_avatar_menu =
-      CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kEnableNewAvatarMenu);
-  bool is_new_profile_management =
-      CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kEnableNewProfileManagement);
-  bool is_consistent_identity =
-      CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kEnableAccountConsistency);
-  bool not_new_avatar_menu =
-      CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kDisableNewAvatarMenu);
-  bool not_new_profile_management =
-      CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kDisableNewProfileManagement);
-  bool not_consistent_identity =
-      CommandLine::ForCurrentProcess()->HasSwitch(
-          switches::kDisableAccountConsistency);
+  bool is_new_avatar_menu = true;
+      /*CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kEnableNewAvatarMenu);*/
+  bool is_new_profile_management = true;
+      /*CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kEnableNewProfileManagement);*/
+  bool is_consistent_identity = true;
+      /*CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kEnableAccountConsistency);*/
+  bool not_new_avatar_menu = false;
+      /*CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kDisableNewAvatarMenu);*/
+  bool not_new_profile_management = false;
+      /*CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kDisableNewProfileManagement);*/
+  bool not_consistent_identity = false;
+      /*CommandLine::ForCurrentProcess()->HasSwitch(
+          switches::kDisableAccountConsistency);*/
   int count_args = (is_new_avatar_menu ? 1 : 0) +
       (is_new_profile_management ? 1 : 0) +
       (is_consistent_identity ? 1 : 0) +
